tools/generator/Tests/Unit-Tests/JavaInteropCodeGeneratorTests.cs

```diff
@@ -49,7 +49,7 @@ public void WriteClassInvokerHandle ()
 		[Test]
 		public void WriteFieldIdField ()
 		{
-			var field = new TestField ("java.lang.String", "bar");
+			var field = new TestField (options, "java.lang.String", "bar");
```

Comment:
...this is the kind of thing I think we want to *avoid*, right? Requiring that `options` be provided to every `*Field` constructor...